test/sample.cc

```diff
@@ -79,19 +81,23 @@ void create_tile() {
   // | sed -e "s/^/#include<cstdint>\n#include<unordered_map>\nstd::unordered_map<size_t,int16_t>
   // pixels {/g" -e "s/$/};/g" > test/pixels.h
   std::vector<int16_t> tile(3601 * 3601, 0);
-  for (const auto& p : pixels)
+  for (const auto& p : pixels) {
     tile[p.first] = p.second;
+  }
+  boost::filesystem::create_directories("test/data/sample/N40");
```

Comment:
i think in the future we are goign to want less boost as we attempt to compile valhalla on more platforms where access to boost (without compiling it from source) will be non existant. that was at least one of the reasons for keeping the directory structure around with `.gitignore` files in them.

Reply:
@kevinkreiser :+1: i have removed `boost::filesystem::create_directories` calls from the PR. `test/data` directory will be created at the configuration stage and sub-directories will be created before running tests.

Reply:
:metal: